specifications/secure-launch-specification.rst

```diff
@@ -479,8 +483,8 @@ environment, that attribute will be published as an entity type. A generic
         u16 entity_type;
         u16 flags;
         u16 reserved;
-        u64 entity;
```

Comment:
This one doesn't change the alignment so asking just to make sure: this is deliberate change and not just overlook, right?

Reply:
I moved size fields around in a number of structures to adjust alignment of fields. I just did it here to to be consistent so it is always size then base addr.